src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/sink/CosmosDBSinkTask.java

```diff
@@ -25,7 +25,7 @@
 public class CosmosDBSinkTask extends SinkTask {
     private static final Logger logger = LoggerFactory.getLogger(CosmosDBSinkTask.class);
     private CosmosClient client = null;
-    private CosmosDBSinkConfig config = null;
+    private CosmosDBSinkConfig config;

```

Comment:
what does this change do?

Reply:
Nothing much really, the configs will be initialized in the start method before any execution happens. Looked at other connectors and saw that they don't assign the config as null.

Reply:
right. it's just sometimes we set things to null explicitly, and sometimes we don't. probably not a big deal.